backend/corpora/common/utils/exceptions.py

```diff
@@ -7,6 +7,26 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
 
+class MaxFileSizeExceededException(CorporaException):
```

Comment:
Introduced application layer exceptions to separate them from HTTP exceptions used for externally facing endpoints.